Lib/test/test_queue.py

```diff
@@ -370,44 +370,35 @@ def _shutdown_all_methods_in_many_threads(self, immediate):
         ev_exec_shutdown = threading.Event()
         res_puts = []
         res_gets = []
-        delay = 1e-4
-        read_process = 4
-        nb_msgs = read_process * 16
-        nb_msgs_r = nb_msgs // read_process
-        when_exec_shutdown = nb_msgs // 2
+        write_threads = 4
+        read_threads = 16
+        nb_msgs = 1024*4
```

Comment:
Seems like quite a few magic numbers here, would also be nice to add a short comment to explain why we choose these

Reply:
Thanks for pointing this out. I recognize that this choice may seem a little arbitrary. The aim is to simulate an example of a multi-producer/consumer queue. When the queue stops, all that's needed are elements in the queue and active producers/consumers. 
Currently, I'm not checking anything about this state.  I'm just checking that only one thread activates the shutdown and that all threads stop cleanly, but nothing about this state.

I set the number of messages (to be put into the queue) very high to make sure I have this context. After running more tests with logs, `read_threads` seems to be too high and `nb_msgs` too low to guarantee this case. I'll modify accordingly.

Reply:
Got it, thank you!